functions/controllers/user/getUsers.js

```diff
@@ -0,0 +1,35 @@
+const { status, message } = require("../../util/constants");
+var HttpStatus = require("http-status-codes");
+
+const getAllUser = async (req, res, db) => {
+    const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = HttpStatus
+    const { error, success } = status
+    const { somethingWentWrong,  } = message
+
+    try {
+        let user = []
+        const users = await db.collection('users').get()
+        if(!users) return res.status(BAD_REQUEST).json({ status: "error", message: somethingWentWrong})
```

Comment:
```suggestion
        if(!users) return res.status(BAD_REQUEST).json({ status: error, message: somethingWentWrong})
```
Use the constant instead of string